code/datums/status_effects/rogue/roguebuff.dm

```diff
@@ -460,6 +472,121 @@
 	desc = span_notice("I am inspired to create!")
 	icon_state = "malum_buff"
 
+
+/*-----------------\
+|   Inhumen Miracles |
+\-----------------*/
+
+/datum/status_effect/buff/call_to_slaughter
+	id = "call_to_slaughter"
+	alert_type = /atom/movable/screen/alert/status_effect/buff/call_to_slaughter
+	duration = 2.5 MINUTES
+	effectedstats = list("strength" = 1, "endurance" = 2, "constitution" = 1)
+
+/atom/movable/screen/alert/status_effect/buff/call_to_slaughter
+	name = "Call to Slaughter"
+	desc = span_bloody("LAMBS TO THE SLAUGHTER!")
+	icon_state = "call_to_slaughter"
+
+
+#define BLOODRAGE_FILTER "bloodrage"
+
+/atom/movable/screen/alert/status_effect/buff/graggar_bloodrage
+	name = "BLOODRAGE"
+	desc = "GRAGGAR! GRAGGAR! GRAGGAR!"
+	icon_state = "bloodrage"
+
+/datum/status_effect/buff/bloodrage
+	id = "bloodrage"
+	alert_type = /atom/movable/screen/alert/status_effect/buff/graggar_bloodrage
+	var/outline_color = "#ad0202"
+	duration = 15 SECONDS
+
+/datum/status_effect/buff/bloodrage/on_apply()
+	var/holyskill = owner.get_skill_level(/datum/skill/magic/holy)
+	duration = ((15 SECONDS) * holyskill)
+	var/filter = owner.get_filter(BLOODRAGE_FILTER)
+	if(!filter)
+		owner.add_filter(BLOODRAGE_FILTER, 2, list("type" = "outline", "color" = outline_color, "alpha" = 60, "size" = 2))
+	if(holyskill >= SKILL_LEVEL_APPRENTICE)
+		effectedstats = list("strength" = 2)
+	else
+		effectedstats = list("strength" = 1)
+	. = ..()
+	return TRUE
```

Comment:
```suggestion
/datum/status_effect/buff/bloodrage/on_creation(mob/living/new_owner, duration_override, ...)
	var/holyskill = owner.get_skill_level(/datum/skill/magic/holy)
	duration = ((15 SECONDS) * holyskill)
	if(holyskill >= SKILL_LEVEL_APPRENTICE)
		effectedstats = list(STATKEY_STR = 2)
	else
		effectedstats = list(STATKEY_STR = 1)
	return ..()

/datum/status_effect/buff/bloodrage/on_apply()
	. = ..()
	owner.add_filter(BLOODRAGE_FILTER, 2, outline_filter(2, outline_color)))
```
additionally there is no reason for outline color to be a variable or to use defines for the filters really, you can use the status effect ID instead